src/libs/gltf/gltf_view/lv_gltf_view_internal.h

```diff
@@ -25,6 +25,7 @@
  *      DEFINES
  *********************/
 
+#define LV_GLTF_AUTOMATIC_DISTANCE_SCALE_FACTOR 2.5
```

Comment:
Note the `f` suffix, also the name can be simplified
```suggestion
#define LV_GLTF_DISTANCE_SCALE_FACTOR 2.5f
```

Reply:
2.5 without any f suffix is what was there originally, a double converted to a float implicitly.  I'll change it to 2.5f if you want.

Reply:
Yes I know, since we're changing it let's do it properly

Reply:
changed in [f33a736](https://github.com/lvgl/lvgl/pull/9143/commits/f33a7362d1aedf02e55c9ff6c75462482ac2abef)